app/pages/project/classify.cjsx

```diff
@@ -92,6 +94,16 @@ module.exports = React.createClass
   componentWillUnmount: () ->
     @context.geordi?.forget ['workflowID']
 
+  componentWillReceiveProps: (nextProps) ->
+    @shouldWorkflowAssignmentPrompt(nextProps.preferences)
+
+  shouldWorkflowAssignmentPrompt: (preferences) ->
+    # Only for Gravity Spy which is assigning workflows to logged in users
+    assignedWorkflowID = preferences?.settings?.workflow_id
+    if @props.project.experimental_tools.indexOf 'workflow assignment' > -1 and @props.user?
```

Comment:
This `indexOf` doesn't have the parentheses around `'workflow assignment'`, but it seems to be working fine, so I'm not sure if the weird Coffeescript and indexOf compiling thing is actually happening. Maybe the `if assignedWorkflowID?` conditional below is stopping it from causing problems?